examples/plot_plotting_metrics_with_error.py

```diff
@@ -0,0 +1,215 @@
+y_test_pred = classifier.predict(X_test_processed)
+
+# %%
+# MetricFrame
+# ===========
+# Now we create a :class:`fairlearn.metrics.MetricFrame` to generate the recall Wilson bounds and recall normal error
+#
+
+# Analyze metrics using :class:`fairlearn.metrics.MetricFrame`
+metrics_dict = {
+    'Recall': recall_score,
+    'Recall Bounds': recall_wilson,
+    'Recall Error': recall_normal_err,
+    'Accuracy': accuracy_score,
+    'Accuracy Bounds': accuracy_wilson,
+    'Accuracy Error': accuracy_normal_err,
+}
+metric_frame = MetricFrame(metrics_dict, y_test_true, y_test_pred, sensitive_features=test_set_sex)
+
+# %%
+# Plotting
+# ========
+# Plot metrics with (symmetric) error bars
+# ----------------------------------------
+plot_metric_frame(metric_frame, kind="scatter",
+                  metrics=['Recall', 'Accuracy'],
+                  error_bars=['Recall Error', 'Accuracy Error'],
+                  figsize=(12, 4))
+plot_metric_frame(metric_frame, kind="bar", metrics='Recall', error_bars='Recall Error')
+
+# %%
+# Plot metrics with confidence intervals (possibly asymmetric)
+# ------------------------------------------------------------
+plot_metric_frame(metric_frame, kind="bar", metrics=[
+                  'Recall', 'Accuracy'], conf_intervals=['Recall Bounds', 'Accuracy Bounds'])
+plot_metric_frame(metric_frame, kind="scatter",
+                  metrics='Recall', conf_intervals='Recall Bounds')
+
+# %%
+# Plot metrics without error bars
+# -------------------------------
+plot_metric_frame(metric_frame, kind="scatter", metrics=['Recall', 'Accuracy'])
+
+# %%
+# Plots all columns and treats them as metrics without error bars
+# ---------------------------------------------------------------
+plot_metric_frame(metric_frame, kind="bar")
+
+# %%
+# Custom Plot
+# -----------
+# Demonstrates how to customize the axes and then pass into `plot_metric_frame`
+
+fig, axs = plt.subplots(*(1, 2), squeeze=False)
+axs = axs.flatten()
+
+axs[0].set_title("Recall Plot")
+axs[1].set_title("Accuracy Plot")
+axs[0].set_xlabel("Sensitive Feature")
+axs[1].set_xlabel("Sensitive Feature")
+axs[0].set_ylabel("Recall")
+axs[1].set_ylabel("Accuracy")
+
+# Set the y-scale for both metrics to [0, 1]
+axs[0].set_ylim((0, 1))
+axs[1].set_ylim((0, 1))
+
+axs = plot_metric_frame(metric_frame,
+                        metrics=['Recall', 'Accuracy'],
+                        error_bars=['Recall Error', 'Accuracy Error'],
+                        figsize=(12, 4),
+                        axs=axs,
+                        kind="bar",
+                        colormap="Pastel1",
+                        rot=45)
+
+# %%
+# Creating Custom Error Metrics
+# =============================
+# In this tutorial we used Normal intervals and Wilson bounds to demonstrate how to plot symmetric and asymmetric metrics, respectively.
+#
+# However, in order to support many different metrics, we reuse the MetricFrame functionality by writing a function
```

Comment:
Please use ReST directives like
```
:class:`fairlearn.metrics.MetricFrame`
```
here and everywhere else.
Similarly, wrap `y_true` and other variables:
```
:code:`y_true`
```

Reply:
Yes, I've highlighted some, but all need to be updated.